venv/lib/python3.12/site-packages/anyio/to_process.py

```
@@ -0,0 +1,258 @@
+    # Redirect standard streams to os.devnull so that user code won't interfere with the
+    # parent-worker communication
+    stdin = sys.stdin
+    stdout = sys.stdout
+    sys.stdin = open(os.devnull)
+    sys.stdout = open(os.devnull, "w")
+
+    stdout.buffer.write(b"READY\n")
+    while True:
+        retval = exception = None
+        try:
+            command, *args = pickle.load(stdin.buffer)
```

Ruff (0.13.3)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)